### DISCUSSION TITLE
NGINXUNIT

### DISCUSSION THREAD

**Discussion by LfJohnVo** [serversideup/docker-php#342] [Q&A] [answered]
Hello everyone, its there guide to how correctly configure nginunit unit on docker and nginx as reverse proxy? :D

**Comment by jaydrogers** [comment]
Not behind NGINX, but we do use Traefik a lot.

I have to update this project to the new v3 beta, but most of the configuration should be the same: 

https://github.com/serversideup/financial-freedom/blob/775f25d206a6bbea829d06bfc65a79df93336ef1/docker-compose.dev.yml#L3-L19

I know it's not what you're exactly looking for, but it's my best shot 😃

**Reply by LfJohnVo** [reply]
Thanks!!! do you have a guide to correctly use your nginxunit image to any project? :D

**Reply by jaydrogers** [reply, marked as answer]
Not yet! Hoping to get more time to make more content now that I am wrapping up an open source building marathon 😆
